Scroll Glitch

noumantahir opened the pull request:
### What does this PR?
handling hyperlink image press using custom image component

### Screenshots/Video
https://user-images.githubusercontent.com/6298342/165411696-3e822575-a756-46ea-aba5-578b1623eeae.mov

![Screenshot 2022-04-27 at 4 57 55 AM](https://user-images.githubusercontent.com/6298342/165411892-e8fc3536-7341-4b48-b8dd-9b3fef53b0ef.png)

### Blocker
This PR attempts handle glitch only when hyperlink have one child and that is an image. However glitch is still there when rendering other hyperlinks as they are still handled by Render Html component as shown in demo video as well.

The issue is, we do not know how complex can a hyper link get, if it contains custom text and content with either deep or multiple children, it will become very difficult to process them.

That being said, the issue is very minimal in my opinion and is something that would occur rarely, the image one however was quite considerable and can be slightly improved as well.

@feruzm what are your thoughts on this.

**Blocker Update**
We should be able to handle this type of no children hyperlink as well, possible issue is we can lose parent font properties passing on to hyperlink.

<img width="1004" alt="Screenshot 2022-04-27 at 5 08 46 AM" src="https://user-images.githubusercontent.com/6298342/165413044-8ac8b30b-04ac-4167-aa0e-b3458c452cb9.png">

If we use onPress method of <Text the glitch does not go away, but if we wrap it in some touchable, the glitch go away however we lose text integrity, check screenshot below.... spend about 30 minutes but could not figure it out so far.

<img width="273" alt="Screenshot 2022-04-27 at 5 46 03 AM" src="https://user-images.githubusercontent.com/6298342/165416048-8665c2a9-0343-4e15-8333-1acd349c685d.png">

feruzm commented:
@noumantahir I am not sure, mobile app having some limits in functionality is ok I think but I don't to complicate the logic if it is hard to maintain and improve in future. I am ok with leaving issue as is for now.

noumantahir commented:
@feruzm  **Update:**
- Added slight activeOpacity for anchored image to differentiate between regular and anchored image
- Dropped deep processing raw hyperlinks for now to avoid glitch for now

https://user-images.githubusercontent.com/6298342/165868657-ed99357a-4e75-445a-a3a4-0e02db5e58a9.mov

feruzm reviewed (approved):
Lgtm
